data/accountManager.go

```diff
@@ -56,14 +56,19 @@ func MakeAccountManager(log logging.Logger, registry account.ParticipationRegist
 	return manager
 }
 
-// Keys returns a list of Participation accounts.
-func (manager *AccountManager) Keys(rnd basics.Round) (out []account.Participation) {
+// Keys returns a list of Participation accounts, and their keys/secrets for requested round.
+func (manager *AccountManager) Keys(rnd basics.Round) (out []account.ParticipationRecordForRound) {
 	manager.mu.Lock()
 	defer manager.mu.Unlock()
 
 	for _, part := range manager.partKeys {
```

Comment:
`manager.partKeys` is the account managers' cache of the legacy old partkey files. Since you're switching to the participation registry it may be good to delete that object and switch everything over to `manager.registry.GetAll()`.

Reply:
Yes it does seem to be no longer relevant since the Participation data is now contained in the registry, I'll remove it completely.

Reply:
I've noticed that `DeleteOldKeys()` is dependent on this `PersistedParticipation` cache, and removing it completely means the ephemeral voting keys can no longer be deleted from the old database. It basically means we remove most logic from `oldKeyDeletionThread` and leave only `DeleteExpired` which is implemented in the registry.
We should implement deleting old voting keys in the registry and probably remove the old database entirely, but it seems out of scope for this feature branch.
So I think we should keep the `partKeys` for now. 
Thoughts? @winder @cce

Reply:
@Aharonee that sounds reasonable, thanks!

Reply:
@winder so it's OK to be looping over `part := range manager.partKeys` and mixing it with a call to `manager.registry.GetForRound(part.ID(), rnd)`? you seemed concerned earlier that it was mixing accessing partKey cache with the ParticipationRegistry.

Reply:
We should still switch to `GetAll()`, but no need to fully remove `partKeys`

Reply:
I'll use the `registry.GetAll()` wherever I can instead of `partKeys` but keep it for compatibility in `DeleteOldKeys`, until we have that fully implemented within the registry.